packages/mobile/src/apollo/types.ts

```diff
@@ -1,54 +1,328 @@
-import { Query } from 'react-apollo'
-import { CURRENCY_ENUM } from 'src/geth/consts'
-import {
-  TransactionStatus,
-  TransactionTypes,
-  TransferTransactionTypes,
-} from 'src/transactions/reducer'
-
-export interface UserTransactionsVariables {
-  address: string
+/*
+ * THIS FILE HAS BEEN GENERATED. DO NOT EDIT IT DIRECTLY.
+ *
+ * REGENERATE WITH:
+ * `yarn run build:gen-graphql-types`
+ */
+
+export type Maybe<T> = T | null
+
+/** All built-in and custom scalars, mapped to their actual values */
+export interface Scalars {
+  ID: string
+  String: string
+  Boolean: boolean
+  Int: number
+  Float: number
+  Address: string
+  Timestamp: number
+  Decimal: string
+  Upload: any
 }
 
-export interface ExchangeTransaction {
-  type: TransactionTypes.EXCHANGE
-  id?: string
-  hash?: string
-  inValue: number
-  outValue: number
-  inSymbol: CURRENCY_ENUM
-  outSymbol: CURRENCY_ENUM
-  timestamp: number
-  status?: TransactionStatus
+export enum CacheControlScope {
+  Public = 'PUBLIC',
+  Private = 'PRIVATE',
 }
 
-export interface TransferTransaction {
-  type: TransferTransactionTypes
-  id?: string
-  hash?: string
-  value: number
-  symbol: CURRENCY_ENUM
-  timestamp: number
-  address: string
-  comment: string
-  status?: TransactionStatus
-  // TODO: fee needs to be added here
+export type Event = Exchange | Transfer
```

Comment:
I guess you kept the old Event interface for backwards compatibility with older app versions? But the next time we ship the app, we'll be resetting the network too and forcing users to update. We could actually remove all that.

Reply:
Indeed! Let's remove it then.

Reply:
I'll remove the deprecated code after this PR is merged to prevent breaking things unnecessarily given `blockchain-api` has to be redeployed.